config/config.js

```diff
@@ -27,25 +27,20 @@ var getGlobbedPaths = function(globPatterns, excludes) {
         if (urlRegex.test(globPatterns)) {
             output.push(globPatterns);
         } else {
-            glob(globPatterns, {
-                sync: true
-            }, function(err, files) {
-                if (excludes) {
-                    files = files.map(function(file) {
-                        if (_.isArray(excludes)) {
-                            for (var i in excludes) {
-                                file = file.replace(excludes[i], '');
-                            }
-                        } else {
-                            file = file.replace(excludes, '');
+            var files = glob.sync(globPatterns);
+            if (excludes) {
```

Comment:
Isn’t the excludes array here being treated as a rename operation instead of an ignore pattern.

For instance. If the cwd contained
public/lib/a
public/lib/b

`glob.sync(‘**’, {ignore: ‘public’})` would return [‘public/lib/‘, ‘public/lib/a’, ‘public/lib/b’]

Whereas `getGlobbedPaths(‘**’, [‘public/‘])` would return [‘lib/‘, ‘lib/a’, ‘lib/b’]

In this case the glob `ignore` option couldn’t be used.